azure-pipelines.yml

```diff
@@ -12,4 +12,4 @@ pr:
 jobs:
   - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
+#  - template: ./.azure-pipelines/azure-pipelines-win.yml
```

Comment:
Please revert the changes to the CI scripts.

Reply:
Ok. But how I remove the `win` task?

The recipes, and packages, are `noarch` but the `win` task fails at the very begining.

Reply:
Failing `noarch` packages on Windows are OK, we can merge them with a red cross.